### ISSUE TITLE
[ios] Missing accessibility content labelling

### ISSUE THREAD

**Issue by kirylkaveryn** [organicmaps/organicmaps#7865]
This issue is related to the same on Android https://github.com/organicmaps/organicmaps/issues/6638

It needs to cover the UI elements with accessibility descriptions.

**Comment by biodranik** [comment]
@kirylkaveryn is there a way to detect/list all missing hints? Maybe even add their detection into an automated CI check?

**Comment by biodranik** [comment]
We can close https://github.com/organicmaps/organicmaps/issues/5894 ?

**Comment by kirylkaveryn** [comment]
> @kirylkaveryn is there a way to detect/list all missing hints? Maybe even add their detection into an automated CI check?

Accessibility hints/labels should be directly set into an _every_ UI element such as views/buttons/textviews/labels etc. In the other case, VoiceOver will use the "text" property to vocalize.
There are no accessibility labels in the ios for now. So we need to add accessibilityLabel to the every button without text.

It is extremely big feature to implement and should be manually tested. Because its important to understand which element can be directly tapped/swiped/visible by the user and skip others (View usually can be complex and contains a lot other view and labels).
